conf/inventory/rhel-8.3-server-x86_64-medlarge.yaml

```diff
@@ -0,0 +1,42 @@
+version_id: 8.3
+id: rhel
+instance:
+  create:
+    image-name: rhel-8.3-server-x86_64-released
+    vm-size: ci.m1.medlarge
```

Comment:
any particular reason  for this `ci.m1.medlarge` size

Reply:
This allows us to reduce the VM footprint but the hard truth is that the memory footprint has not reduced to the expected extended. For sanity, I believe we can keep the footprint small.

Reply:
so can you remove this inventory file, we can use the existing rhel-8.3 inventory file since other tests will also added in this suite soon.

Reply:
@rakeshgm I don't remember if I got you convinced on using the newer inventory file during our side discussions.